include/wasmtime.hh

```diff
@@ -1556,8 +1556,9 @@ public:
    * This function is the same as wasmtime_module_deserialize except that it
    * reads the data for the serialized module from the path on disk. This can
    * be faster than the alternative which may require copying the data around.
-   * the artifacts of a previous compilation to quickly create an in-memory
-   * module ready for instantiation.
+   *
+   * This function does not take ownership of any of its arguments, but the
+   * returned error and module are owned by the caller.
```

Comment:
Oh I think that while comments about ownership are necessary for C with C++ I was hoping that comments about ownership aren't necessary since it's theoretically evident in the types.

Reply:
Ah yes, makes sense, I (poorly) copy/pasted from the C API comments.